azure-local/upgrade/install-solution-upgrade-azure-resource-manager-template.md

```diff
@@ -0,0 +1,270 @@
+
+11. On the **Review + Create** tab, select **Create**. This creates the remaining prerequisite resources and validates the upgrade. Validation takes about 10 minutes to complete.
+
+    :::image type="content" source="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-create.png" alt-text="Screenshot showing Create selected on Review + Create tab." lightbox="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-create.png":::
+
+12. Once validation is complete, select **Redeploy**.
+
+    :::image type="content" source="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-redeploy.png" alt-text="Screenshot showing Redeploy selected." lightbox="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-redeploy.png":::
+
+13. On the **Custom deployment** screen, select **Edit parameters**. Load up the previously saved parameters and select **Save**.
+
+14. Change the final value in the JSON from **Validate** to **Deploy**, where **Deployment Mode = Deploy**.
+
+    :::image type="content" source="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-deploy.png" alt-text="Screenshot showing deploy selected for deployment mode." lightbox="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-deploy.png":::
+
+15. Verify that all the fields for the ARM template are filled in by the parameters JSON.
+
+16. Select the appropriate resource group for your environment.
+
+17. Confirm that **Deployment Mode = Deploy**.
+
+18. Select **Review + create**.
+
+19. Select **Create**. The upgrade begins, using the existing prerequisite resources that were created during the **Validate** step.
+
+    The Deployment screen cycles on the cluster resource during upgrade.
+
+    Once the upgrade initiates, there's a limited Environment Checker run, a full Environment Checker run, and cloud upgrade starts. After a few minutes, you can monitor upgrade in the portal.
+
+    :::image type="content" source="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-environment-checker.png" alt-text="Screenshot showing the status of environment checker validation." lightbox="./media/install-solution-upgrade-azure-resource-manager-template/deploy-arm-template-environment-checker.png":::
+
+20. In a new browser window, navigate to the resource group for your environment. Select the cluster resource.
+
+21. Select **Deployments**.
+
+22. Refresh and watch the deployment progress from the first machine (also known as the seed machine and is the first machine where you deployed the cluster). Deployment takes between 2.5 and 3 hours. Several steps take 40-50 minutes or more.
+
+## ARM template parameters reference
+
```

Comment:
If we are planning to add this parameters reference, definitely we should use an include. Another option is to put ARM template parameters reference in a standalone and link out to it.